modules/gapi/include/opencv2/gapi/own/exports.hpp

```diff
@@ -8,11 +8,18 @@
 #ifndef OPENCV_GAPI_OWN_TYPES_HPP
 #define OPENCV_GAPI_OWN_TYPES_HPP
 
+#   define GAPI_WRAP
```

Comment:
Shall it be mapped to `CV_WRAP` in case of `#   if defined(__OPENCV_BUILD)` ? Similar to `GAPI_EXPORTS_*`

Reply:
Yes, fixed